### ISSUE TITLE
No more buttons to minimize/maximize/close window

### ISSUE THREAD

**Issue by mrichez-saluc** [squirrel-sql-client/squirrel-sql-code#22]
I'm using Squirrel 4.6 on Arch Linux, there are no buttons to minimize/maximize/close window.

![2023-09-14_09-18](https://github.com/squirrel-sql-client/squirrel-sql-code/assets/60512254/7615715f-d697-4045-853d-ac9fe100b00f)

**Comment by tsdh** [comment]
What window manager or wayland compositor are you using?  For example, I'm using sway and see the same as your screenshot but I'd consider it correct.  The compositor adds the window decorations (borders, titlebar, and buttons) or maybe not (if it's a tiling compositor like sway), not the application itself.

There is a trend for client-side decorations (CSD) where the application is responsible for drawing its decorations itself.  For example, most Gnome applications or Chrome/Electron-based applications do.  Java applications, Emacs, Firefox, and many others don't.  I guess that's the thing confusing you.

Basically, what you want is to tell your window manager/compositor to draw window decorations for some apps (like Squirrel) but not for others.  Right now, you seem to have disabled them completely.

**Comment by mrichez-saluc** [comment]
My window manager is X11.

**Comment by tsdh** [comment]
X11 is no window manager.  Something like Gnome, KDE, Xmonad, i3wm is.

**Comment by mrichez-saluc** [comment]
So, it's Gnome :-)

**Comment by mrichez-saluc** [comment]
Don't understand how, but it's now resolved... sorry for noise
